src/__namespace.js

```diff
@@ -1,4 +1,5 @@
 /**
  * @ignore
  */
-wikibase.datamodel = wikibase.datamodel || {};
+window.wikibase = window.wikibase || {};
```

Comment:
Could some JS wizard explain what is the different before the code before and after? Would both work in all cases? What "normal" browsers do different so that it apparently was not an issue before.

This comment is not mean as anything against this particular PR. I just don't understand intricacies of Java Script.